openpathsampling/snapshot.py

```diff
@@ -117,306 +91,73 @@ def copy(self):
         content (e.g. Configuration object) will not.
 
         """
-        this = AbstractSnapshot(is_reversed=self.is_reversed, topology=self.topology)
+        this = self.__class__.__new__(self.__class__)
+        AbstractSnapshot.__init__(this, topology=self.topology)
         return this
 
-    def reversed_copy(self):
-        """
-        Returns a shallow reversed copy of the instance itself. The
-        contained configuration and momenta are not copied and the momenta
-        are marked reversed.
-
-        Since Snapshots are made in pairs this will also lead to a new
-        (non-)reversed copy!
-
-        Returns
-        -------
-        Snapshot()
-            the reversed shallow copy
-        """
+    def create_reversed(self):
+        this = self.copy()
+        this._reversed = self
+        return this
 
-        obj = self.copy()
-        return obj.reversed
 
+class FeatureSnapshot(AbstractSnapshot):
```

Comment:
But isn't the use of snapshot features optional in either case? I mean, can I get the behavior of features while subclassing from `AbstractSnapshot`?

It seems like non-feature-snapshots are deprecated now... Normally we might keep deprecated aspects around a bit to give users a chance to change, but right now our only users are the people over at dynamiq-md -- and I'm pretty sure they can keep up with the changes.